src/main/resources/portal.properties.EXAMPLE

```diff
@@ -71,7 +71,8 @@ skin.study_view.link_text=To build your own case set, try out our enhanced Study
 authorization=false
 ## enable and set this property to specify a study group to be used to identify public studies for which no specific authorization entries are needed in the `authorities` table
 # always_show_study_group=
-## which method of authentication to use (false, googleplus, saml, openid, ad, ldap, noauthsessionservice)
+## which method of authentication to use (false, public_auth, googleplus, saml, openid, ad, ldap, noauthsessionservice)
+## when set to public_auth(having authentication for public and social-google-auth), make sure to set googleplus.consumer.key and googleplus.consumer.secret
```

Comment:
Again, "social_auth" not "public_auth".  microsoftlive.consumer.key and microsoftlive.consumer.secret should be added to this comment too.

Reply:
You also should add:

`# required even if not using social_auth`
`microsoftlive.consumer.key=`
`microsoftlive.consumer.secret=`

Without this an exception is thrown and the application will not start up, even if authorization=false and authenticate=false.  I am not sure why this is the case, it is not true of other keys defined in applicationContext-security.xml.  An alternative is to modify applicationContext-security.xml so that you use default values (by adding a ":"), e.g.

`${microsoftlive.consumer.key:}`
`${microsoftlive.consumer.secret:}`

That seems to make it happy, although that might not be a good idea.  Probably better to just define them in this file so at least the users have to think about what they are doing.